volumes/volume.go

```diff
@@ -146,14 +146,14 @@ func (v *Volume) FromDisk() error {
 }
 
 func (v *Volume) jsonPath() (string, error) {
-	return v.getRootResourcePath("config.json")
+	return v.GetRootResourcePath("config.json")
 }
-func (v *Volume) getRootResourcePath(path string) (string, error) {
```

Comment:
@cyphar would you mind adding just a small documentation for GetRootResourcePath and GetResourcePath, even if it's just "go look at contianer's doc"